approach.md

```diff
@@ -3,8 +3,28 @@ layout: page
+Our approach begins with engaging communities actively researching the wide-ranging issues we face in content publishing today, learning from independent media that are practicing alternative models of publishing and governance, as well as progressive technology standards and experimental platforms.
+Our project team of creators and technologists collaborate in a tight feedback loop, constantly integrating our respective perspectives into the initiative to ensure the tool we build will serve creators.
 
-If you are interested in having a conversation about possible ways we may collaborate, [please get in touch](mailto:hello@distributed.press)!
\ No newline at end of file
+## Phase 1: Learning about the ecosystem
+
+Beginning this year, we interviewed founders of progressive platforms like [Popula](https://popula.com) and [Are.na](http://are.na), activists and journalists who combat misinformation and censorship, as well as proponents and builders of open protocols from the IndieWeb and DWeb movement.
+The discussions covered a lot of areas relating to content publishing, from tensions between publicity and anonymity, struggles with tools and infrastructures of authorship and distribution, personal experiences of censorship, learnings from experiments of collective governance, value trade-offs of a decentralized future in a world of misinformation, particular technical implementations, and increasingly dominant publishing platforms that undervalue quality while pushing creators toward atomization and precarious labor.
+Many of the interview notes are published on [GitHub](https://github.com/hyphacoop/distributed.press/blob/master/interviews.md).
+These interviews also surfaced many relevant initiatives we were previously unaware of, which prompted us to explore new territories and to engage in new discussions. Some of these groups will become collaborators in the next phase of our project.
+
+We would like to recognize badpennie, SoapDog, Emmi Bevensee, Dawn Walker, Maria Bustillos, Jack Jamieson, and others who wish to stay anonymous, for their invaluable input at this phase of the _Distributed Press_ project, as well as Ace Volkov for his contribution of illustrations on our project website.
+Since these interviews, our project team has discussed our learnings extensively, synthesizing the information internally and with our collaborators, to propose the next phase of our project.
+
+## Phase 2: Building an author-owned magazine on the DWeb
+
+We're creating [COMPOST](https://compost.digital), a magazine about the digital commons that provides fertile ground for creative experimentation, cultivates communities that value participation, and celebrates the labor of regeneration and care.
+
+In recognition that quality content is often the work of sustainable teams of authors, editors, artists, and photographers, the tool we build for creators must enable reliable livelihoods and democratic governance.
```

Comment:
In recognition that quality content is often the work of diverse teams of authors, editors, artists, and photographers, the tool we build must enable reliable livelihoods and democratic participation for all.